content/_releases/22/14.md

```diff
@@ -0,0 +1,267 @@
+
+
+### Flow deletion job
+
+Introducing a new cron job called `clear-deleted-flows` which permanently deletes
+flows from the MongoDB with the `DELETED` status. Every user trying to delete a
+flow will see a [warning message](#flow-deletion-times) informing that this is a
+permanent and irreversible action.
+
+#### HELM3 optional variables
+
+We introduce new environment variables which you can use to control the
+`clear-deleted-flows` job:
+
+*   `WIPER_CLEAR_DELETED_FLOWS_AGE_SECONDS` - Time in seconds the job must wait before deleting the flow permanently after it is marked as `DELETED` in the MongoDB. We set the default value to `86400` seconds (1 day).
+*   `WIPER_CLEAR_DELETED_FLOWS_LIMIT` - Number of flows the `clear-deleted-flows` job will permanently delete each time it runs.
+
+#### Release process
+
+This release process applies if you already have a cluster running previous versions
+of the platform code. **The {{page.number}} release will start an automatic migration.**
+
+*   Backup the MongoDB data base before the deployment to have an opportunity to reverse the process.
+*   Set `WIPER_CLEAR_DELETED_FLOWS_LIMIT` environment variable to an applicable number for your case to perform this operation faster of you have large number of flows in your DB.
+
+> After the migration the `marathonevents`, `requestbins` and `taskstartlogs`
+> deprecated collection will be removed.
+
+### Iron Bank table creation on all Clickhouse instances
+
+New Environment Variables:
+
+*   `IRON_BANK_CLICKHOUSE_NODES` (required) - array of objects with host (required), port, user and password values
+*   `IRON_BANK_CLICKHOUSE_DATABASE` (optional) - ClickHouse database name (default iron_bank)
+
+```yaml
+IRON_BANK_CLICKHOUSE_NODES:
+   - host: "10.0.0.1"
+     port: "8123"
+     user: "default"
+     password: "123"
+   - host: "10.0.0.2"
+     port: "8123"
+     user: "user"
+     password: "1234"
+IRON_BANK_CLICKHOUSE_DATABASE: "iron_bank"
+```
+
+New Iron Bank provision logic
+
+*   If all nodes in `IRON_BANK_CLICKHOUSE_NODES` are not available, error is thrown and iron bank shuts down
+*   If one of the nodes is not available, warning is logged and provision to all other available nodes is executed
+*   Main connection (to which all querying is executed) is set to the first available node from `IRON_BANK_CLICKHOUSE_NODES`
+
+Removed Environment Variables:
+
+`IRON_BANK_CLICKHOUSE_URI`
+
+
+{% endif %}
+
+## Fixed bugs
+
+*   Addressed issue when the platform token refresher service would return `500` status code in case when the `/refresh/[id]` Oauth server would return an error. Now the service reports `4xx` indicating the third party problem.
+*   Fixed the bug connected with the Profile Page usage example returning `401` in case of API-Key regeneration.
+
+## Components
+
+### [Hubspot Component](/components/hubspot/) `1.5.1`
+
+*   **FIXED** the bug in `Get New and Updated Objects Polling` preventing it to extract more than 10K records.
```

Comment:
preventing it from extracting more than 10K records

Reply:
thanks